### ISSUE TITLE
Log var can become negative and explode

### ISSUE THREAD

**Issue by snie2012** [yaringal/multi-task-learning-example#14]
The loss function can optimize in a way that keep decreasing the log_var values, which I observe in my experiments. One simple solution is to do torch.abs(log_var). Any thoughts on how this might affect the formulation of the deductions?

**Comment by YangLeiSX** [comment]
I have encountered the same situation during my training, do you have any better solutions?
